ivadomed/scripts/automate_training.py

```diff
@@ -260,13 +280,17 @@ def automate_training(config, param, fixed_split, all_combin, n_iterations=1, ru
                 new_config["log_directory"] = initial_config["log_directory"] + name
                 config_list.append(copy.deepcopy(new_config))
 
+    if output_dir and not os.path.exists(output_dir):
+        os.makedirs(output_dir)
+    if not output_dir:
+        output_dir = ""
+
     # CUDA problem when forking process
     # https://github.com/pytorch/pytorch/issues/2517
-    mp.set_start_method('spawn')
+    ctx = mp.get_context("spawn")
```

Comment:
Nitpick: I am not sure if here single-quotes would be more appropriate? (Basing myself off of this: https://github.com/neuropoly/spinalcordtoolbox/wiki/Programming%3A-Single-or-double-quotes)?

Reply:
Interesting! I agree with the logic behind using `" "` for strings with text, as that could contain apostrophes etc. I think maybe switching between the two adds some overhead, like it might be easier to just use `" "` for everything?

Reply:
@ahill187 you make a good point, I tend to lean toward the preference of having either all single quote or all double quotes, so just having `" "` could be a good choice. Seems like this is not the case with the SCT project, but given this is a separate project perhaps we can define our own standard for this?

Reply:
Perhaps it would make more sense to stick with just `" "` for the ivadomed project in general. I did notice SCT has a hybrid policy on this, but I tend to have a preference more toward just have one standard, especially with potential refactoring overhead.

Reply:
Yah I agree!